_posts/2016-02-29-avoid-runtime-errors-in-react-with-flow.markdown

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: "Avoid Runtime Errors in React with Flow"
+date: 2016-02-29 18:00:00
+categories: reactjs flow javascript development
```

Comment:
Later we may want to update the taxonomy to leverage tags in addition to categories so we can keep cats high-level (e.g. development, design, business intelligence) while allowing tags to be used for, keyword relevance and a [posts by tag page](http://habd.as/tags/), for example.